orion/pkg/repos/migrations/000010_add_published_at_table_programs.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE programs DROP COLUMN published_at;
```

Comment:
could you also add `AFTER deleted_at;`

like in here?
https://github.com/ahsu1230/mathnavigatorSite/pull/104/files